### PR TITLE
Remove tooltip on click from like button

### LINKED ISSUE #5497: Tooltip doesn't disappear
**Description**'
Tooltip doesn't disappear when I click on like button. 
Peek 2019-04-17 00-50

This is a bug that needs to be fixed. 

Thanks!

### PR DESCRIPTION
**Pull request by gautamig54**
Fixes #5497  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by divyabaid16** [comment]
Can you add a GIF for this @gautamig54 
Thanks!

**Comment by gautamig54** [comment]
@divyabaid16 Here you go!
Also, since the trigger for tooltip is set to be hover therefore it will close as soon as the mouse pointer leaves the like button.

ezgif com-video-to-gif

**Comment by divyabaid16** [comment]
Awesome @gautamig54 :smile:

File: app/views/like/_like.html.erb [hunk 1]
```diff
@@ -91,7 +91,10 @@
   </style>
 
   <script>
- 
+    $('[rel="tooltip"]').on('click', function () {
+    $(this).tooltip('hide')
```
Comment (CleverFool77) [inline app/views/like/_like.html.erb]:
```suggestion
        $(this).tooltip('hide')
```
Indent 4 spaces.

**Comment by gautamig54** [comment]
@publiclab/reviewers Kindly review this PR. Thanks!

**Comment by sashadev-sky** [comment]
confirmed! So what this change does is it only shows the tooltip the first time the user hovers over the button, and then after that it won't show it again unless you reload. Previously, the tooltip just never went away until you left the page. Please see 2 gifs below demonstrating this. I think this change is fine and mergable! 

**Before:**
tooltip

**After:**
tooltip2

**Comment by sashadev-sky** [comment]
@gauravano @jywarren please review for merge

**Comment by jywarren** [comment]
Fantastic!!!!!!

**Pull request merged**